src/lib/Timer.js

```diff
@@ -0,0 +1,54 @@
+var marky
```

Comment:
this file should not be a design-time artifact but be compiled along w/ the other .ts files

Reply:
can you take care of this, please?